rpm/ovis-ldms-toss4.spec.in

```diff
@@ -257,6 +262,39 @@ package.
 %{_includedir}/*/*/*.h
 %{_includedir}/ovis-ldms-config.h
 
+%if %{with genders}
+%package genders
+Summary: LDMS base initscripts for libgenders control of %{name}
+Group: %{ldms_grp}
+Requires: ovis-ldms = %{version}
+Obsoletes: ovis-initscripts-base
```

Comment:
These are internal SNL packages? Maybe add comment about when they can be removed (2025, whatever).